src/ui/components/common/nav.rs

```diff
@@ -1,213 +1,151 @@
-                                  </A>
-                                </li>
-                                <li>
-                                  <details>
-                                    <summary>"User name"</summary>
-                                    <ul>
-                                      <li>
-                                        <A href="/u/jimmy90">{t!(i18n, nav.profile)}</A>
-                                      </li>
-                                      <li>
-                                        <A href="/settings">{t!(i18n, nav.settings)}</A>
-                                      </li>
-                                      <li>
-                                        <hr/>
-                                      </li>
-                                      <li>
-                                        <ActionForm action=logout_form_action>
-                                          <input
-                                            name="is_ssr"
-                                            type="hidden"
-                                            value=format!("{}", is_ssr.get())
-                                          />
-                                          <button type="submit">{t!(i18n, nav.logout)}</button>
-                                        </ActionForm>
-                                      </li>
-                                    </ul>
-                                  </details>
-                                </li>
-                              }
-                          }
-                      })
-              }}
+            >
 
-            </ErrorBoundary>
-          </Suspense>
+              <li>
+                <A href="/inbox">
+                  <span title=t!(i18n, nav.unread_messages)>
+                    <Icon icon=Icon::from(ChIcon::ChBell) class="h-6 w-6"/>
+                  </span>
+                </A>
+              </li>
+              <li>
+                <details>
+                  <summary>
+                    {with!(
+                        | my_user | { let Person { name, display_name, .. } = my_user.as_ref()
+                        .unwrap(); display_name.as_ref().unwrap_or(name).to_string() }
+                    )}
 
-        </ul>
-      </div>
-    </nav>
+                  </summary>
+                  <ul>
+                    <li>
+                      <A href=with!(
+                          | my_user | format!("/u/{}", my_user.as_ref().unwrap().name)
+                      )>{t!(i18n, nav.profile)}</A>
+                    </li>
+                    <li>
+                      <A href="/settings">{t!(i18n, nav.settings)}</A>
+                    </li>
+                    <li>
+                      <hr/>
+                    </li>
+                    <li>
+                      <ActionForm action=logout_action>
+                        <button type="submit">{t!(i18n, nav.logout)}</button>
+                      </ActionForm>
```

Comment:
This button won't submit the form at all when in this details, so logging out is temporarily broken. I will clean this up in a future PR.